examples/with-vue-nuxt/apps/docs/.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
Can we work around using this / do you know what is requiring it? I know some libraries still don't play nice with pnpms module isolation strategy, but if at all possible it would be great to exclude this, or at the very least only hoist the problematic packages.

Reply:
There's no need for that option at the current time,

The file was auto-generated by `turborepo`, I pushed a commit where I just removed that file
>https://github.com/vercel/turbo/pull/6425/commits/21d31b4916e2548dc837e94df6ab893ec410f731

Reply:
Great, thanks!